hack/generate/gen-test-framework.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#############################
+# Make sure that test/test-framework is updated according to the current source code
+#############################
+
+set -ex
+
+source hack/lib/test_lib.sh
+export GO111MODULE=on
```

Comment:
Do we still need `GO111MODULE=on` now that we're using Go 1.13? 

I think this is related to the fact that the test-framework is not a proper Go module. Maybe we need to solve that problem?

Reply:
Currently yes. See: 

```shell
$ operator-sdk generate k8s
FATA[0000] using go modules requires GO111MODULE="on", "auto", or unset. More info: https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#go-modules 
``` 

Tracked the need in : https://github.com/operator-framework/operator-sdk/issues/2182

Reply:
I think the problem here is that the test-framework needs to look more like a real project since we're now calling `operator-sdk` project commands on it. This PR introduces this need. Other usages of `GO111MODULE` have already been removed from the project.

Reply:
We can check if should really still required there. However, without it now will not work.

Reply:
Catcher 👍